README.md

```diff
@@ -109,7 +109,7 @@ invokes cmake commands as needed.
 * Change to the root of the source code directory and build:
 
         cd arqma
-        make cmake-release && cd build/release && make
+        make cmake-release && cd build/Linux/master/release && make
```

Comment:
We did left it in purpouse. We are using branch/host build dir and you can build from one repo couple binaries and all of them will build at separate forders, eg:
./build/Linux/master/release/bin/ either ./build/Linux/release-v0.2/release/bin when you change branch